src/initialize/src/controllers/gitController.ts

```diff
@@ -9,24 +9,109 @@
  *     IBM Corporation - initial API and implementation
  *******************************************************************************/
 
-import git from 'simple-git/promise';
 import fs from 'fs-extra';
 import path from 'path';
+import request from 'request';
+import { URL } from 'url';
 
 import { replacePlaceholderNameWithProjectName } from './replaceController';
+import { exec } from 'child_process';
 
 export async function importProjectFromGitRepo(repo: string, name: string, dest: string, branch?: string | undefined): Promise<void> {
   if (fs.readdirSync(dest).length > 0) {
     throw new Error('Git clone failed - directory to clone into is not empty');
   }
 
+  const tmpName = `/tmp/${branch}_${Date.now()}`;
```

Comment:
Is this path also supported with the windows temp dir?

Reply:
No, we only expect this to run inside a Linux container.